bndtools.core/src/bndtools/central/Central.java

```diff
@@ -461,6 +465,9 @@ public static void refreshPlugins() throws Exception {
                         l.modelChanged(p);
                 }
 
+                if (refreshedRepos.size() > 0) {
+                    repositoriesViewRefresher.repositoriesRefreshed();
```

Comment:
I still think you need to call this in the `refreshPlugin(Refreshable plugin)` method below. No?